source/js/nav-newsletter.js

```diff
@@ -42,8 +42,13 @@ class NavNewsletter {
   // remove the global 'closeFormClickHandler' click event handler
   // and reset the form
   closeDesktopNewsletter(event) {
-    elements.container.classList.remove("expanded");
+    const wrapper = elements.container;
+    wrapper.classList.remove("expanded");
     elements.buttonDesktop.classList.remove("active");
+    setTimeout(() => {
```

Comment:
Assuming this is to wait for the sliding animiation to finish? Can we add inline comment here for future reference?

Reply:
can do

Reply:
```suggestion
    // Schedule a "display:none" to happen after the `expanded` animation finishes.
    // See `#nav-newsletter-form-wrapper` transition in ./source/sass/components/primary-nav.scss 
    setTimeout(() => {
```